hoodie-client/src/main/java/com/uber/hoodie/HoodieWriteClient.java

```diff
@@ -1051,7 +1051,7 @@ private HoodieCommitMetadata commitForceCompaction(JavaRDD<WriteStatus> writeSta
   private void rollbackInflightCommits() {
     HoodieTable<T> table = HoodieTable.getHoodieTable(
         new HoodieTableMetaClient(jsc.hadoopConfiguration(), config.getBasePath(), true), config, jsc);
-    HoodieTimeline inflightTimeline = table.getMetaClient().getCommitsTimeline().filterInflights();
+    HoodieTimeline inflightTimeline = table.getMetaClient().getCommitsTimeline().filterInflightsExcludingCompaction();
```

Comment:
How will rollback work for inflight compactions ?

Reply:
@n3nash This is addressed in one of the stacked diffs